src/components/ConversationSettings/ConversationSettingsDialog.vue

```diff
@@ -50,6 +62,16 @@
 			class="app-settings-section">
 			<MatterbridgeSettings />
 		</AppSettingsSection>
+		<!-- Destructive actions -->
+		<AppSettingsSection
+			v-if="canLeaveConversation || canDeleteConversation"
+			:title="t('spreed', 'Danger zone')"
+			class="app-settings-section">
+			<DangerZone
+				:conversation="conversation"
+				:can-leave-conversation="canLeaveConversation"
```

Comment:
"Danger zone" with only the leave action looks a bit odd. Doesn't feel that dangerous to leave a conversation, there is no actual loss as you can ask to be re-added.

But deleting would delete messages, so it's indeed dangerous.

Reply:
I'd say it's a naming issue, because grouping them is reasonable imo.
let's tackle it here #5253